conf/application.conf

```diff
@@ -57,7 +57,7 @@ logger.play=INFO
 # Logger provided to your application:
 logger.application=DEBUG
 
-assets.defaultCache="public, max-age=60"
+play.assets.defaultCache="public, max-age=60"
```

Comment:
As we don't do hashed asset names, is this caching too aggressive?

Reply:
Fair question. You're probably right, however I'd suggest addressing this is out of scope for this PR? (This diff is to remove a deprecation warning in the logs).

Happy to have a different PR that addresses this go in before this one does.

Reply:
I read that as 2 months and not a minute, so this is fine.